src/HeapDump/CrossGenerationLiveness/CrossGenerationLivenessCollector.cs

```diff
@@ -223,19 +224,28 @@ public static void LoadNative(string relativePath)
         /// <summary>
         /// Get the name of the architecture of the current process.
         /// </summary>
-        private static string ProcessArch
+        private static ProcessorArchitecture ProcessArch
         {
             get
             {
-                if (s_ProcessArch == null)
+                return Environment.Is64BitProcess ? ProcessorArchitecture.Amd64 : ProcessorArchitecture.X86;
```

Comment:
We had some support for ARM ETW collection but even that rotted after the windows team requires special  signing to run things on ARM.   We never made HeapDump work.  It is fine to ignore ARM for the foreseeable future.